packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -693,9 +685,13 @@ const Summary: React.FunctionComponent = () => {
 
 				<div className="focused-launch-summary__ask-for-help">
 					<p>{ __( 'Questions? Our experts can assist.', __i18n_text_domain__ ) }</p>
-					<a href="/help" onClick={ onAskForHelpClick }>
+					<Button
+						isLink
+						href={ localizeUrl( 'https://wordpress.com/help/contact', locale ) }
```

Comment:
Does this URL need to be localised? I thought that logged in pages didn't need to (see p58i-9Mt-p2#comment-48983)

Reply:
In #47947, we implemented i8n for the step-by-step-flow HE support link. Since the purpose of the link is entirely the same in the focused launch flow, I assume that it has to be localized as well. 

@razvanpapadopol, what do you think?

Reply:
Just FYI, according to the A/C, the expected correct link should be `https://wordpress.com/es/help/` — but if I open that link on my browser, I get a [404](https://wordpress.com/es/help/)

Reply:
> the expected localised link should be https://wordpress.com/es/help/

That was a wrong assumption from my side (see explanation below) and I've updated the A/C & Testing instructions.

#### Explanation

We use the `useLocalizeUrl()` method to localize our URLs but some ([a list is provided in the hook](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-utils/src/localize-url.tsx#L81-L102)) don't have to be localized (see [`packages/i18n-utils/src/localize-url.tsx`](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-utils/src/localize-url.tsx#L139-L143)). This is also the case for `wordpress.com/help/contact`. When [a user is logged in](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-utils/src/localize-url.tsx#L95-L97), and this is always the case when entering the Focused Launch Flow, the original URL is returned and not a localized one.